nessie/src/main/java/org/apache/iceberg/nessie/NessieCatalog.java

```diff
@@ -65,6 +65,7 @@ public class NessieCatalog extends BaseMetastoreCatalog
 
   private static final Logger LOG = LoggerFactory.getLogger(NessieCatalog.class);
   private static final Joiner SLASH = Joiner.on("/");
+  public static final String NAMESPACE_LOCATION_PROPS = "location";
```

Comment:
it doesn't look like this is used outside of this class, so let's make this `private`

Reply:
done